README.md

```diff
@@ -1,23 +1,36 @@
 
-| Operating System        | Status       | Latest Binary |
-| ----------------------- |:------------:|:-------------:|
+As with many open source packages, we use [GitHub](https://github.com/analogdevicesinc/libiio) to do develop and maintain the source. If you think you have found a bug in a release, before attempting to debug things too much, try the master branch. We provide some nightly builds for common operating systems. If you need something else, we can most likely add that -- just ask.
+
+| Operating System        | GitHub master status  | Latest (untested) Binary |
+| ----------------------- |:---------------------:|:------------------------:|
 | Windows                 | [![Windows Status](https://ci.appveyor.com/api/projects/status/github/analogdevicesinc/libiio?svg=true)](https://ci.appveyor.com/project/analogdevicesinc/libiio/branch/master) | [![Latest Windows build](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/win_box.png)](https://ci.appveyor.com/api/projects/analogdevicesinc/libiio/artifacts/libiio-setup.exe?branch=master)
-| OS X                    | [![OSX Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=osx&passingTex=foo)](https://travis-ci.org/analogdevicesinc/libiio) |
-| Linux                   | [![Linux Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=linux)](https://travis-ci.org/analogdevicesinc/libiio) |  [![Debian](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/deb_box.svg)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.deb) [![RPM File](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/rpm_box.png)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.rpm)|
+| OS X                    | [![OSX Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=osx&passingTex=foo)](https://travis-ci.org/analogdevicesinc/libiio) | [![OS-X package](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/osx_box.png)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.pkg)
+| Linux                   | [![Linux Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=linux)](https://travis-ci.org/analogdevicesinc/libiio) |  [![Debian](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/deb.png)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.deb) [![RPM File](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/rpm.png)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.rpm)|
 
+If you use it, and like it - please let us know. If you use it, and hate it - please let us know that too. The goal of the project is to try to make Linux IIO devices easier to use on a variety of platforms. If we aren't doing that - we will try to make it better.
 
-Library for interfacing with IIO devices
+Feedback is appreciated (in order of preference):
```

Comment:
I think it would make sense to specify which link is best for which case:
* Github trackers for talking about bugs and improvements,
* EZ for general help on libiio and/or IIO,
* the IIO mailing list for kernel-specific stuff.